stellar_account_prometheus_exporter/exporter.py

```diff
@@ -93,17 +95,27 @@ def do_GET(self):
                 if "num_sponsoring" not in r.json():
                     self.error(500, "Error - no num_sponsoring found for account {}".format(account["account_id"]))
                     return
+                if "subentry_count" not in r.json():
+                    self.error(500, "Error - no subentry_count found for account {}".format(account["account_id"]))
+                    return
 
                 labels = [network["name"], account["account_id"], account["account_name"]]
                 m_num_sponsored.labels(*labels).set(r.json()["num_sponsored"])
                 m_num_sponsoring.labels(*labels).set(r.json()["num_sponsoring"])
 
                 for balance in r.json()["balances"]:
                     labels = [network["name"], account["account_id"], account["account_name"], balance["asset_type"]]
+
                     m_balance.labels(*labels).set(balance["balance"])
                     m_buying_liabilities.labels(*labels).set(balance["buying_liabilities"])
                     m_selling_liabilities.labels(*labels).set(balance["selling_liabilities"])
 
+                    # ref: https://github.com/stellar/stellar-protocol/blob/a664806db12635ab4d49b3f006c8f1b578fba8d4/core/cap-0033.md#reserve-requirement
+                    minimum_required_balance = balance["selling_liabilities"]
+                    if balance["asset_type"] == "native":
+                        minimum_required_balance += 0.5 * (2 + r.json()["subentry_count"] + r.json()["num_sponsoring"] - r.json()["num_sponsored"])
```

Comment:
@jacekn: I changed these lines since the last time you saw the PR because `subentry_count`, `num_sponsoring` and `num_sponsored` don't need to be accounted for when the asset is not XLM.